parameter.c

```diff
@@ -1,6 +1,6 @@
 #include <string.h>
 #include "parameter.h"
-#include <parameter_port.h>
+#include "parameter_port.h"
```

Comment:
The ```<>``` brackets are needed here to be able to overwrite the port header by a project specific one.